pod/perldelta.pod

```diff
@@ -59,6 +59,13 @@ XXX For a release on a stable branch, this section aspires to be:
 
 [ List each incompatible change as a =head2 entry ]
 
+=head2 sort with no arguments is now a compile-time error
+
+C<@a = sort;> used to be equivalent to C<@a = ();>, which wasn't very
+useful; it is now a compile-time error. The old behaviour has been
+removed to make room for future features. Sorting an empty array (ie
+C<@empty = (); @a = sort @empty;>) continues to be valid syntax.
+
```

Comment:
S<> should probably be added.

I have no substantive issues with this P.R.  LGTM